tests/Autofix/FixerCollectionTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Nusje2000\ComposerMonolith\Tests\Autofix;
+
+use Nusje2000\ComposerMonolith\Autofix\FixerCollection;
+use Nusje2000\ComposerMonolith\Autofix\FixerInterface;
+use PHPUnit\Framework\TestCase;
+
+final class FixerCollectionTest extends TestCase
+{
+    public function testConstruct(): void
+    {
+        $fixers = [
+            $this->createStub(FixerInterface::class),
+        ];
+
+        $collection = new FixerCollection($fixers);
+
+        self::assertSame($fixers, $collection->toArray());
```

Comment:
You're testing the `Aeviiq\Collection\ObjectCollection` here. I'd say this test should be in that library, not in yours.

Reply:
I added this because it was never used in tests but since the introduction of the FixerTest this became redundant. Ill remove it.